lib/install/filter-invalid-actions.js

```diff
@@ -8,13 +8,25 @@ module.exports = function (top, differences, next) {
   validate('SAF', arguments)
   var action
   var keep = []
+
+  differences.forEach(function (action) {
+    var cmd = action[0]
+    var pkg = action[1]
+    if (cmd === 'remove') {
+      pkg.removing = true
+    }
+  })
+
   /*eslint no-cond-assign:0*/
   while (action = differences.shift()) {
     var cmd = action[0]
     var pkg = action[1]
-    if (pkg.isInLink || pkg.parent.target) {
-      log.warn('skippingAction', 'Module is inside a symlinked module: not running ' +
-        cmd + ' ' + getPackageId(pkg) + ' ' + path.relative(top, pkg.path))
+    if (pkg.isInLink || pkg.parent.target || pkg.parent.isLink) {
+      // we want to skip warning if this is a child of another module that we're removing
```

Comment:
A brief note of _why_ we're skipping the warning would be helpful here (e.g. "because it's very noisy and gratuitous")